packages/trader/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -115,11 +117,10 @@ class AccountSignup extends React.Component {
                                             <Button
                                                 className={classNames('account-signup__btn', { 'account-signup__btn--disabled': !values.residence || errors.residence })}
                                                 type='button'
-                                                is_disabled={ !values.residence || errors.residence }
+                                                is_disabled={ !values.residence || !!errors.residence }
```

Comment:
may i know why are we using double negation here ?

Reply:
Because it's passing the `string` for `errors.residence` instead of `bool`. The `Button`'s component `is_disabled` property only accepts `boolean`, if not we get spammed with validation warning errors when not deploying to staging environment.